File: Programación y Algoritmos/Actividad 15/estatura.cpp
#include <iostream>
#include <locale.h>

using namespace std;

int contador = 0;
int total = 0; 

int main() 
{
	setlocale(LC_ALL, "");

	cout << "Vaya introduciendo la estatura en centimetros de los 5 alumnos" << endl;
	for (int i = 1; i <= 5; i++)
	{
		
		int estatura = 0 ;
		while (estatura < 110 || estatura > 220)
		{
			cout << "Escribe la estatura del alumno " << i << ": ";
			cin >> estatura; 
		}
		if (estatura > 170)
		{
			contador++;
		}
		total = total + estatura;


	}

	cout << "El promedio de estatura de los alumnos es: " << total/5 << "cm" << endl;
	cout << "Hubo " << contador << " alumnos mas altos que 170cm" << endl;

	
}

File: Programación y Algoritmos/Actividad 15/estatura.py
print("Introduzca consecutivamente las estaturas de los 5 alumnos en centímetros")


total = 0
contador = 0
for i in range(5):
    estatura = 0
    while estatura < 110 or estatura > 220:
        estatura = int(input("Introduzca la estatura del alumno " + str(i + 1) + ": "))

    total += estatura
    if estatura > 170:
        contador += 1
    
print(f"El promedio de estatura es de {total/5}cm")
print(f"Hay {contador} alumnos más altos que 170")